### DISCUSSION TITLE
A 6 channel relay box with XIAO

### DISCUSSION THREAD

**Discussion by 256ericpan** [Seeed-Studio/OSHW-XIAO-Series#8] [In Development]
For smart building upgrade

**Comment by 256ericpan** [comment]
https://www.flux.ai/ericspan/8-relay-controller-for-xiao?editor=pcb_3d on going

**Reply by 256ericpan** [reply]
we are doing it on livestream session, if you want a sample, please comment below and DM me your seeed bazar account or address. (first come, first serve)

**Reply by 256ericpan** [reply]
https://www.youtube.com/watch?v=B5nNPyg5IO4

**Reply by gmacario** [reply]
> we are doing it on livestream session, if you want a sample, please comment below and DM me your seeed bazar account or address.

I missed the livestream today (it was 4 AM local time :sob:) - am I still on time for the sample offer?
